packages/vuetify/src/components/VColorPicker/VColorPickerPreview.tsx

```diff
@@ -35,6 +42,18 @@ export const VColorPickerPreview = defineComponent({
   },
 
   setup (props, { emit }) {
+    const supportEyePicker = 'EyeDropper' in window
+
+    async function openEyePicker(){
+      const eyeDropper = new EyeDropper()
+      try{
+        const result = await eyeDropper.open();
```

Comment:
Should probably be passed AbortSignal when the component unmounts

Reply:
Done.